src/components/HeaderWithSearchBar/HeaderWithSearchBar.vue

```diff
@@ -7,6 +7,23 @@
         v-if="logoUrl"
       />
       <h2 class="header-with-search-bar__title" v-else>{{ title }}</h2>
+      <q-btn
+        class="header-with-search-bar__cart-btn"
+        @click="$router.push({ name: 'shoppingCart' })"
+        size="10px"
+        round
+        color="accent"
+        v-if="showShoppingCart"
+      >
+        <shopping-cart-icon :width="15.67" :height="19.76" />
```

Comment:
Please check if these measures are fine for different viewports

Reply:
Notice the button says 10px but the icon have 19.76

Reply:
It seems that `size="10px"` refers to font-size.

The size of the button is `width: 2.75rem; height: 2.75rem;`.